src/sql/src/parser.rs

```diff
@@ -471,4 +482,54 @@ mod tests {
             })
         );
     }
+
+    #[test]
+    pub fn test_explain() {
+        let sql = "EXPLAIN select * from foo";
+        let result = ParserContext::create_with_dialect(sql, &GenericDialect {});
+        let stmts = result.unwrap();
+        assert_eq!(1, stmts.len());
+
+        let select = sqlparser::ast::Select {
```

Comment:
Just a comment, another way is to compare the formatted result string, but this style is also good 👍